ch_tools/chadmin/internal/zookeeper.py

```diff
@@ -340,10 +354,49 @@ def databases_cleanups(zk, zk_root_path):
         _delete_recursive(zk, hosts_paths)
         _remove_if_no_hosts_in_replicas(zk, databases_paths)
 
+    def get_host_mention_in_task(zk, host, ddl_task_path):
+        """
+        Predicate for indicating that host must execute this task.
+
+        If there is a host mention in the ddl, then returns the escaped hostname with port.
+        """
+        try:
+            for line in zk.get(ddl_task_path)[0].decode().strip().splitlines():
```

Comment:
Using https://github.com/yandex/ch-tools/blob/0ba8de49db877e2f087ff7ddec69f2c4fe9fbcd7/ch_tools/chadmin/internal/zookeeper.py#L15-L19 will make the code more readable imho

Reply:
Think that it is hard to reuse this method, because we already have formatted paths and opened zk connection